corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -48,6 +48,7 @@
             $('#commcare-settings').koApplyBindings(COMMCAREHQ.appView.settings);
         });
 
+        {% if not app.is_remote_app %}
```

Comment:
Currently JS breaks while viewing remote app deploy page because `#multimedia` div doesn't exist for remote app